City-of-Helsinki/drupal-helfi-sote
# UHF-1333 Drupal configuration

How to test:
- `git checkout UHF-1333_drupal_configuration`
- `make new`
- Check that the site works and check that the site name has been updated https://helfi-sote.docker.sh/en/admin/config/system/site-information
